### PR TITLE
warn about CVE-2020-1967

### PR DESCRIPTION
**Pull request by lzutao**

### CODE REVIEW

File: crates/openssl-src/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "openssl-src"
+date = "2020-04-25"
+title = "Invalid or unrecognised signature algorithm is received from the peer"
+url = "https://www.openssl.org/news/secadv/20200421.txt"
+categories = ["crypto-failure", "denial-of-service"]
```
Comment (alex) [inline crates/openssl-src/RUSTSEC-0000-0000.toml:7]: I don't think it's correct to say this is a crypto-failure. While the bug happens in a crypto protocol, there's no actual issue with the cryptograpy.

Comment (lzutao) [reply crates/openssl-src/RUSTSEC-0000-0000.toml]: my mistake.

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2020-0015` in #286

**Comment by MikailBag** [comment]
Does it affect `openssl` crate users? If it does, which functions are affected?

**Comment by lzutao** [comment]
Yes if users use the vendored feature: https://github.com/sfackler/rust-openssl/blob/e446d819e32fb9e9e1debf645b7f741c125a76e5/openssl/Cargo.toml#L19

**Pull request merged**